dotnet-client-libraries/ITextMergingPDFs/ITextMergingPDFs/Merger.cs

```diff
@@ -41,17 +42,17 @@ public string[] SplitDocument(string pdfFile)
 
         private string GetFullFileName(string mergedPdfFileName)
         {
-            return Path.Combine(_path, mergedPdfFileName);
+            return System.IO.Path.Combine(_path, mergedPdfFileName);
         }
 
         private static string[] CheckParameters(string[] pdfFiles, string mergedPdfFileName)
         {
             var existingFiles = pdfFiles.Where(File.Exists).ToArray();
 
-            if (existingFiles is null || existingFiles.Length == 0)
+            if (existingFiles.Length == 0)
                 throw new ArgumentOutOfRangeException(nameof(pdfFiles));
 
-            if (string.IsNullOrEmpty(mergedPdfFileName))
+            if (string.IsNullOrWhiteSpace(mergedPdfFileName))

```

Comment:
This should probably be the first check in the method - the fail fast principle, as before this we are doing work, walking an array and then allocating a new one.

Reply:
Repaired